src/App.tsx

```diff
@@ -2,13 +2,23 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+import { PostList } from './components/PostList';
+
+const preparedPosts = posts.map(post => {
+  const userLink = users.find(user => user.id === post.userId) || null;
+  const commentLink = comments.filter(comment => comment.postId === post.id) || null;
```

Comment:
избегай слово link
в JS по-другому не бывает, все объекты и массивы ссылки

```suggestion
  const user = users.find(user => user.id === post.userId) || null;
  const comments = comments.filter(comment => comment.postId === post.id) || null;
```